x-pack/plugins/cloud_security_posture/common/schemas/csp_rule_metadata.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema as rt, TypeOf } from '@kbn/config-schema';
+
+export const ruleMetadata = rt.object({
+  audit: rt.string(),
+  benchmark: rt.object({ name: rt.string(), version: rt.string() }),
+  default_value: rt.maybe(rt.nullable(rt.string())),
```

Comment:
So currently there's an issue with the integration, setting it as null when there is no value.
And for backward compatibility I guess we should use undefined (when the field does not exists yet)

Also the same interface is being used by the `findings`, and there I think it is undefined instead of null (the way `cloudbeat` actually sends the data)

Reply:
TBH I didn't explored that in such depth.. I can give it more time and dig into the differences. But I don't think it worth the time.. let me know what you think

Reply:
Since this is **the** specification of our data model, I suggest diving deep and getting the most concise representation of the rule object